.github/workflows/tag-deploy-app-test.yaml

```diff
@@ -185,14 +185,14 @@ jobs:
         fetch-depth: 0
 
     # Navigate to frontend dir, install dependencies, execute cypress command
-    - name: Setup and Execute Cypress e2e test suite
+    - name: Setup and Execute WebDriverIO e2e test suite
       shell: bash
       env:
-        TEST_IDIR_PASSWORD : ${{ secrets.TEST_IDIR_PASSWORD }}
-      run: |
-          cd ./src/frontend
-          npm install
-          CYPRESS_PASSWORD=$TEST_IDIR_PASSWORD npm run cy:e2e-test
+          mpoPass : ${{ secrets.TEST_IDIR_PASSWORD }}
```

Comment:
Same as above

Reply:
Thanks Brandon! Corrected the same with d3c9041e260b1359ac12f5d9aecc2ed0bb515d0e